### PR TITLE
Export Wins ETL Task

### PR DESCRIPTION
**Pull request by marcuspp**

### CODE REVIEW

File: app/etl/tasks/export_wins_country.py [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+    case
+        when s.standardised_country is not null then s.standardised_country
+        else d.country
+    end as country,
+    d.sector as sector,
+    'export_win' as type,
+    '{data_sources.export_wins}' as source,
+    d.export_wins_id::text as source_id,
+    d.date_won::timestamp as timestamp
+from {ExportWins.get_fq_table_name()} d
```
Comment (dvoong) [inline app/etl/tasks/export_wins_country.py]: unrelated but do you know what fq means? Found the definition here, but still not sure what the name means

https://github.com/uktrade/countries-of-interest-service/blob/develop/app/db/models/__init__.py#L43

Comment (marcuspp) [reply app/etl/tasks/export_wins_country.py]: @kristof-bb `fully qualified` ?

File: app/etl/tasks/export_wins_country.py [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+        d.export_wins_company_id::text as service_company_id,
+        null::int4 as company_match_id,
+        case
+            when c.name is not null then c.name
+            when s.standardised_country is not null then s.standardised_country
+            else NULL
+        end as country,
+        d.sector as sector,
+        'export_win' as type,
+        'export_wins' as service,
```
Comment (marcuspp) [inline app/etl/tasks/export_wins_country.py]: Do you think this is right? `type` = `export_win` `service` = `export_win(s)` or should they be both `export_wins`

File: app/etl/__init__.py [hunk 3]
```diff
@@ -1,32 +1,39 @@
...
+
+import sqlalchemy.exc
+from flask import current_app
+
+logger = logging.getLogger(__name__)
+logger.setLevel(logging.INFO)
 
 
 class ETLTask:
+    name = ''
```
Comment (kristof-bb) [inline app/etl/__init__.py]: might be cleaner to define this as an abstract class property using ABCMeta to make clear it needs to be implemented in subclasses, but guess its fine for now.

**Pull request merged**